Makefile

```diff
@@ -59,25 +59,25 @@ js-packages/browser/package.json: package.json
 js-packages/webworker/package.json: package.json
 	clojure -T:build sync-package-json :target $(@D)/package.json
 
-js-packages/nodejs/flureenjs.js: out/flureenjs.js
+js-packages/nodejs/fluree-node-sdk.js: out/fluree-node-sdk.js
 	cp $< $@
 
-js-packages/browser/flureedb.js: out/flureedb.js
+js-packages/browser/fluree-browser-sdk.js: out/fluree-browser-sdk.js
 	cp $< $@
 
 js-packages/webworker/flureeworker.js: out/flureeworker.js
```

Comment:
Should we rename these to `fluree-web-worker.js` or similar?

Reply:
I was planning to, but I think the current webworker is broken so I didn't want to mess with it too much until we get the actual artifact working.

Reply:
I would personally still vote to sync up its naming convention in here, but I'll leave it up to you. At some point it was working, but I don't remember how long ago that was and I don't think it gets much testing. So I wouldn't be shocked if you were right.